src/StringExporter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/*
+ * This file is part of the Exporter package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SebastianBergmann\Exporter;
+
+/**
+ * Exporter for visualizing strings.
+ *
+ * @package    Exporter
+ * @author     Sebastian Bergmann <sebastian@phpunit.de>
+ * @copyright  Sebastian Bergmann <sebastian@phpunit.de>
+ * @license    http://www.opensource.org/licenses/BSD-3-Clause  The BSD 3-Clause License
+ * @link       https://github.com/sebastianbergmann/exporter
+ */
+class StringExporter extends BaseExporter
+{
+    /**
+     * Returns whether the exporter can export a given value.
+     *
+     * @param  mixed   $value The value to export.
+     * @return boolean
+     */
+    public function accepts($value)
+    {
+        return is_string($value);
+    }
+
+    /**
+     * Recursively exports a value as a string.
+     *
+     * @param  mixed                              $value       The value to export
+     * @param  integer                            $indentation The indentation level of the 2nd+ line
+     * @param  SebastianBergmann\Exporter\Context $processed   Contains all objects and arrays that have previously been rendered
+     * @return string
+     * @see    SebastianBergmann\Exporter\Exporter::export
+     */
+    protected function recursiveExport(&$value, $indentation, $processed = null)
```

Comment:
implementing this as ``recursiveExport`` looks wrong to me, as it is not recursive at all). IMO, this should be implemented as ``export`` directly (without extending BaseExporter, as you don't really need any code from it, but implementing an interface used by any of the exporters)